### DISCUSSION TITLE
AudioResources.dat cannot be found

### DISCUSSION THREAD

**Discussion by GuichardVictor42** [publicsamples/Oi-Grandad#13] [Q&A]
Hey, I downloaded Oi! Grandad today when I saw it in a Venus Theory video, but when trying to load the plugin into Ableton Live, the interface shows up but is all blurred and I get the following error message: 

"The file C:\Users\victo\AppData\Roaming\modularsamples\oi grandad\AudioResources.dat couldn't be found."

When downloading, I just put the unzipped folder '"oi.grandad.vst3.win" that only contains the file "oi grandad.vst3" in the folder by which Ableton accesses all the VST3s on my computer (which is just C:\VST3 Installations in my case), so I don't really know where I should find this additionnal .dat file. Could you please help me with this ?

Thanks in advance and have a great day !

V. Guichard

**Comment by OneShortMoment** [comment]
Unfortunately the installation instructions are a bit vague...
After some trial and error, here's what worked for me:

1. On the releases page where you downloaded the oigrandand.vst3.win.zip file, there's another file (linked from the description at the top) - it says "Presets & Samples", but it also contains the missing AudioResources.dat  -> you'll need that
2. After unpacking that second file, copy its contents (a folder called modularsamples) to c:\users\<your_username>\AppData\Roaming\
(So, if your Windows user would be "Otto", you now should have a folder called C:\Users\Otto\AppData\Roaming\modularsamples - containing a bunch of subfolders and files, among them the AudioResources.dat file.)
3. If you now insert the plugin into your DAW again, it should work (and also contain a few examples / factory presets)

**Reply by RickTaylor79** [reply]
> Unfortunately the installation instructions are a bit vague... After some trial and error, here's what worked for me:
> 
> 1. On the releases page where you downloaded the oigrandand.vst3.win.zip file, there's another file (linked from the description at the top) - it says "Presets & Samples", but it also contains the missing AudioResources.dat  -> you'll need that
> 2. After unpacking that second file, copy its contents (a folder called modularsamples) to c:\users<your_username>\AppData\Roaming
>    (So, if your Windows user would be "Otto", you now should have a folder called C:\Users\Otto\AppData\Roaming\modularsamples - containing a bunch of subfolders and files, among them the AudioResources.dat file.)
> 3. If you now insert the plugin into your DAW again, it should work (and also contain a few examples / factory presets)

Thanks for stepping in here, I haven't been getting notifications for these posts. 

The latest release fixes the AudioResources.dat requirement, meaning the plugin can be used independently of the presets folder.
